src/dataflow/src/logging/materialized.rs

```diff
@@ -27,7 +27,7 @@ use repr::{Datum, Row, Timestamp};
 pub type Logger = timely::logging_core::Logger<MaterializedEvent, WorkerIdentifier>;
 
 /// A logged materialized event.
-#[derive(Debug, Clone, Ord, PartialOrd, Eq, PartialEq, Hash)]
+#[derive(Debug, Clone, PartialOrd, PartialEq)]
```

Comment:
hmm why did this change?

Reply:
It's kinda a remnant of the metrics-as-logging-tables change: I needed to pass things that don't impl these traits (f64, ugh), and realized that these traits are entirely unused & don't need to be there, so removed them. I can rebase that away if necessary, but the next person who needs something like an f64 on a message field might be happy it's gone (:

Reply:
thats fine then